Makefile

```
@@ -44,3 +44,12 @@ validate:
 	@echo "+++ validating config $(CONFIG)"
 	@echo
 	go run ./cmd/hpsf -i $(CONFIG) validate
+	for format in rConfig rRules cConfig ; do \
+		go run ./cmd/hpsf -i $(CONFIG) $${format} || exit 1; \
+	done
```

```suggestion
	@echo
	@echo "+++ generating configs for $(CONFIG)"
	@echo
	for format in rConfig rRules cConfig ; do \
		echo "\n+++ generating config for $${format}\n"; \
		go run ./cmd/hpsf -i $(CONFIG) $${format} || exit 1; \
	done
```
I love this! Wonder if we can add some extra "helper text" line to convey what each piece is doing.

---

added some text around this 👍🏻